### DISCUSSION TITLE
Sticky navbar won't work over divs

### DISCUSSION THREAD

**Discussion by coderr7511** [twbs/bootstrap#36200] [Q&A]
Hello,
Can anyone help?
I've tried to implement a sticky navbar into my website, it worked over one section however over a section where there are many divs it no longer appears. Any suggestions? 
<img width="382" alt="Screen Shot 2022-04-20 at 8 33 48 pm" src="https://user-images.githubusercontent.com/104069446/164212548-c41d1d8b-9990-4742-8cae-2819bee0a838.png">
<img width="269" alt="Screen Shot 2022-04-20 at 8 34 47 pm" src="https://user-images.githubusercontent.com/104069446/164212701-b536e8dc-d21c-478a-b238-900930af8f74.png">
<img width="1006" alt="Screen Shot 2022-04-20 at 8 37 16 pm" src="https://user-images.githubusercontent.com/104069446/164213060-2b9503e5-3aab-42b4-b175-628dddfdda10.png">

**Comment by gerardnico** [comment]
Why are you not using the placement class ?

https://getbootstrap.com/docs/5.0/components/navbar/#placement
